plugins/woocommerce-admin/client/inbox-panel/test/index.js

```diff
@@ -22,43 +25,75 @@ jest.mock( '@wordpress/data', () => {
 	};
 } );
 
-jest.mock( '@woocommerce/tracks', () => ( { recordEvent: jest.fn() } ) );
+jest.mock( '@woocommerce/experimental', () => {
+	// Require the original module to not be mocked...
+	const originalModule = jest.requireActual( '@woocommerce/experimental' );
+
+	return {
+		__esModule: true, // Use it when dealing with esModules
+		...originalModule,
+		InboxNoteCard: jest.fn().mockImplementation( ( { note } ) => {
+			return <div>{ note.id }</div>;
+		} ),
```

Comment:
Question from before still stands... why do we need to mock this?

Reply:
It's because I don't know how to use the original on all the other test cases besides the one from line 169. 

There is no explicit need to mock it. I just couldn't make it work in any other way. Do you know if there's a way?

Reply:
Ah, that makes sense. Hmm.. I would think you could just mock `@woocommerce/experimental` in the specific test where you need it mocked, but I'm not positive. If you want to leave it like this, that's cool.